src/DefaultRenderer.js

```diff
@@ -211,6 +211,11 @@ export default class DefaultRenderer extends Component {
     }
 
     if (SceneComponent) {
+      if (navigationState.tabs || !navigationState.parent) {
```

Comment:
What is the need for having a different code path here?  Is there something in the Example app I can check?  Does this change break any of the Example app pages?